src/util/substitute.h

```diff
@@ -1,20 +0,0 @@
-/*******************************************************************\
```

Comment:
substitute is used in TG

Reply:
Is there a reason not to use `std::replace`?

Reply:
Moved to #2214.